packages/core/redux/persistAccount/actions.js

```diff
@@ -18,6 +18,9 @@ import {
 } from '@chronobank/core/redux/persistAccount/utils'
 import networkService from '@chronobank/login/network/NetworkService'
 import profileService from '@chronobank/login/network/ProfileService'
+import web3Provider from '@chronobank/login/network/Web3Provider'
+import web3Utils from '@chronobank/login/network/Web3Utils'
+import mnemonicProvider from '../../../login/network/mnemonicProvider'
```

Comment:
PLease use '@chronobank/login' instead of '../../../login'. In other case it'll brake ChronoMint-RN